io.read_nwb issues with new nwb release (now pinned to pynwb<1.4)

nwb 1.4.0 was released a couple of hours ago and io.save_nwb is broken. Test shows the following
```
suite2p/io/nwb.py:80: in read_nwb

    F = np.array(nwbfile.processing['ophys']['Fluorescence']['roi_response_series'].data)
>               raise KeyError(msg)

E               KeyError: "'roi_response_series' not found in Fluorescence 'Fluorescence'."

.eggs/hdmf-2.1.0-py3.7.egg/hdmf/container.py:663: KeyError
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -40,7 +40,7 @@ jobs:
 install:
 - conda env create -f environment.yml
 - source activate suite2p
-- pip install .[data]
+- pip install .[data,nwb]
 - dvc pull -r gdrive-travis
 - pip install coveralls
 script:

--- a/setup.py
+++ b/setup.py
@@ -28,7 +28,6 @@ setuptools.setup(
     ],
     tests_require=[
       'pytest',
-      'pynwb',
     ],
     extras_require={
       "docs": [
@@ -53,6 +52,9 @@ setuptools.setup(
         "dvc>=1.1",
         "pydrive2",
       ],
+      "nwb": [
+        "pynwb<1.4",
+      ]
     },
     include_package_data=True,
     classifiers=[

